x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -15,301 +15,276 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
+    documentation:
+      'Relative URL from which to download global artifact manifests. Default: /downloads/endpoint/manifest/artifacts-<version>.zip.',
   },
   {
     key: 'windows.advanced.artifacts.global.public_key',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Public key used to verify the global artifact manifest signature.',
   },
   {
     key: 'windows.advanced.artifacts.global.interval',
     first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'windows.advanced.artifacts.user.base_url',
-    first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'windows.advanced.artifacts.user.ca_cert',
-    first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Interval between global artifact manifest download attempts, in seconds. Default: 3600.',
   },
   {
     key: 'windows.advanced.artifacts.user.public_key',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Public key used to verify the user artifact manifest signature.',
   },
   {
     key: 'windows.advanced.artifacts.user.interval',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Interval between user artifact manifest download attempts, in seconds. Default: 3600.',
   },
   {
     key: 'windows.advanced.elasticsearch.delay',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Delay for sending events to elasticsearch, in seconds. Default: 120.',
   },
   {
     key: 'windows.advanced.elasticsearch.tls.verify_peer',
     first_supported_version: '7.11',
-    documentation: 'default is true',
+    documentation: 'Whether to verify the certificates presented by the peer. Default: true.',
   },
   {
     key: 'windows.advanced.elasticsearch.tls.verify_hostname',
     first_supported_version: '7.11',
-    documentation: 'default is true',
+    documentation:
+      "Whether to verify the hostname of the peer is what's in the certificate. Default: true.",
   },
   {
     key: 'windows.advanced.elasticsearch.tls.ca_cert',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Text blob of a certificate authority.',
   },
   {
     key: 'windows.advanced.malware.quarantine',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Whether quarantine should be enabled when malware prevention is enabled. Default: true.',
   },
   {
     key: 'windows.advanced.ransomware.mbr',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      "A value of 'false' overrides other config settings that would enable ransomware MBR. Default: true.",
   },
   {
     key: 'windows.advanced.ransomware.canary',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      "A value of 'false' overrides other config settings that would enable ransomware canary. Default: true.",
```

Comment:
@magermark @gabriellandau should I change this to `Default: false.` per https://github.com/elastic/endpoint-dev/blob/5f7127a8dbf4850c71b65b0cf42ac1359df9c000/Elastic/Plugins/Policy/Lib/plugins/Ransomware.cpp#L67 ? Again, I assumed it would be changed to `true` by 7.11 release.

Reply:
Canary files will `not` be incorporated into the 7.11 silent release, so `Default: false` would make sense in this case for the time being until the 7.11 release. Still not sure how ransomware strictly being a silent feature for 7.11 affects these potential schema changes, though.

Reply:
Also Ransomware should be removed here for now.

Reply:
removed in ffa65db31caa5a5bca2ad19868587d115a8ffe04